app/models/pipeline_run.rb

```diff
@@ -34,8 +34,6 @@ class PipelineRun < ApplicationRecord
   STATUS_READY = 'READY'.freeze
   POSTPROCESS_STATUS_LOADED = 'LOADED'.freeze
 
-  before_create :create_run_stages
```

Comment:
did you do a scan on PipelineRun to make sure there's only one place for create_run_stages?

Reply:
i did

but i now realize i don't need to change this